src/main/resources/application.yaml

```diff
@@ -80,7 +80,7 @@ spring:
         registration:
           oidc:
             client-id: microservicekey-am-role-assignment-service
-            client-secret: ${AM_ROLE_ASSIGNMENT_SERVICE_SECRET:AAAAAAAAAAAAAAAA}
+            client-secret: ${AM_ROLE_ASSIGNMENT_SERVICE_SECRET:}
```

Comment:
you also need to remove default value from below statement.These should be fetch from vault.
    username: ${ORG_ROLE_MAPPING_DB_USERNAME:am}
    password: ${ORG_ROLE_MAPPING_DB_PASSWORD:am}
    sharedAccessKeyValue: ${AMQP_SHARED_ACCESS_KEY_VALUE:test}